iOS4-HW21-ArtemMuho.playground/Sources/WorkThread.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+public class WorkThread: Thread {
+    var store: Store
+    
+    public init(store: Store) {
+        self.store = store
+    }
+    
+    public override func main() {
+        Timer.scheduledTimer(timeInterval: 1, target: self, selector: #selector(remove), userInfo: nil, repeats: true)
+        while !self.store.store.isEmpty {
+            self.store.remove()
+            print("Микросхема припаяна")
+        }
```

Comment:
У `scheduledTimer` есть более удобный метод с использованием замыкания:
```suggestion
        Timer.scheduledTimer(withTimeInterval: 1, repeats: true) { _ in
            while !self.store.store.isEmpty {
                self.store.remove()
                print("Микросхема припаяна")
            }
        }
```
Если оставить как у тебя, то здесь происходит дублирование кода, в таком случае можно просто вызвать функцию `remove()`